packages/account/src/Sections/Verification/Helpers/verification.js

```diff
@@ -16,12 +16,13 @@ export const populateVerificationStatus = account_status => {
     const is_idv_disallowed = account_status.status.some(status => status === 'idv_disallowed');
     const poa_address_mismatch = account_status.status.some(status => status === 'poa_address_mismatch');
     const poi_expiring_soon = account_status.status.some(status => status === 'poi_expiring_soon');
+    const poa_authenticated_with_idv = account_status.status.some(status => status === 'poa_authenticated_with_idv');
     const poa_expiring_soon = account_status.status.some(status => status === 'poa_expiring_soon');
 
     const has_poa = !(document && document.status === 'none');
     const has_poi = !(identity && identity.status === 'none');
     const has_poinc = !(income && income.status === 'none');
-    const has_submitted_poa = document_status === 'pending' && !allow_poa_resubmission;
+    const has_submitted_poa = document_status === 'pending' && !allow_poa_resubmission && !poa_authenticated_with_idv;
```

Comment:
This will effect non -p2p users as well. isnt?

Reply:
I'll remove it since this condition is not needed. All users will have to go through the same process of poa approval once submitted.